pages/en/lb4/Mixin.md

```diff
@@ -0,0 +1,119 @@
+---
+lang: en
+title: 'Mixin'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Mixin.html
+summary:
+---
+
+It is a commonly used JavaScript/TypeScript strategy to extend a class with new properties and methods.
+
+*question: does our mixinBuilder allow interface extends classes?*
+
+## Mixin Builder
+
+LoopBack provides a mixin builder function `MixinBuilder` to easily extend a class 
+with one or multiple mixins,
+you can call it to define the new extended class with fluent syntaxed API like:
+
+```ts
+import {MixiBuilder} from '@loopback/repository';
+import {BaseClass} from '<path_exports_BaseClass>';
+import {FooMixin, BarMixin} from '<path_exports_Mixins>';
+
+let newClass = MixinBuilder.mix(baseClass).with(FooMixin, BarMixin);
+```
+
+## Define Mixin
+
+By defining a mixin, you create a mixin function that takes in a base class, 
+and returns a new class extends the base class with new properties and methods mixed to it.
```

Comment:
perhaps: "returns a new class extends the base class.." --> "returns a new class extending the base class..."